src/predict/api/controllers/prediction.py

```diff
@@ -26,6 +29,7 @@ def get_next_24hr_forecasts():
 
 
 @ml_app.route(api.route['next_1_week_forecasts'], methods=['GET'])
+@cache.cached(timeout=Config.CACHE_TIMEOUT, key_prefix=weekly_forecasts_cache_key)

```

Comment:
Thanks @Mnoble-19 , has this been updated in the SECRETS and what not for all ENVs?

Reply:
Yes. I've updated secret manager